### PR TITLE
Include reverse wards check in `wards.sh` + refactoring

### PR DESCRIPTION
**Pull request by naszam**
- Include reverse wards check case in `wards.sh` (Target -> Contract)
  - Tested with `End` as target
    - `make wards target=MCD_END`
    - https://hackmd.io/@naszam/SJWRD0sI9
 - Refactor `src` wards check
   - Tested `src` wards check with `0x1f42e41A34B71606FcC60b4e624243b365D99745` as target

### CODE REVIEW

**Comment by brianmcmichael** [comment]
```[[ "$1" ]] || { echo "Please specify the Target Address or ChainLog Key (ASCII) to inspect"; exit 1; }```

Consider: `"Please specify the Target Address (make target=0xaddress...) or ChainLog Key (make target=KEY) to inspect"`

File: scripts/wards.sh [hunk 1]
```diff
@@ -3,7 +3,18 @@ set -e
 
 [[ "$ETH_RPC_URL" ]] || { echo "Please set a ETH_RPC_URL"; exit 1; }
```
Comment (brianmcmichael) [inline scripts/wards.sh:4]:
```suggestion
[[ "$ETH_RPC_URL" ]] || { echo "Please set a ETH_RPC_URL"; [ -v PS1 ] && return || exit; }
```

Avoids closing the terminal window when running the script directly.

Comment (brianmcmichael) [reply scripts/wards.sh:4]: Can also use on next line.

File: scripts/wards.sh [hunk 2]
```diff
@@ -3,7 +3,18 @@ set -e
...
 
-[[ "$1" ]] || { echo "Please specify the Target Address or ChainLog Key (ASCII) to inspect"; exit 1; }
+[[ "$1" ]] || { echo "Please specify the Target Address (e.g. target=0xB966002DDAa2Baf48369f5015329750019736031) or ChainLog Key (e.g. target=MCD_VAT) to inspect"; exit 1; }
```
Comment (brianmcmichael) [inline scripts/wards.sh:6]:
```suggestion
[[ "$1" ]] || { echo "Please specify the Target Address (e.g. target=0xB966002DDAa2Baf48369f5015329750019736031) or ChainLog Key (e.g. target=MCD_VAT) to inspect"; [ -v PS1 ] && return || exit; }
```

File: scripts/wards.sh (hunk 1 above)
Comment (brianmcmichael) [reply scripts/wards.sh:4]: Avoids closing the terminal window when running the script directly (e.g. `. ./scripts/wards.sh`)

Comment (naszam) [reply scripts/wards.sh:4]: I've reverted latest changes for now, as discussed in chat, to investigate more on an approach to address this issue.

**Review by brianmcmichael — APPROVED** [review]
LGTM

**Pull request merged**